### ISSUE TITLE
Add opt-in/op-out option for users.

### ISSUE THREAD

**Issue by RuudschMaHinda** [nextcloud/impersonate#61]
- I think it would be great if a user can opt-in to impersonate being used on their account. Maybe limit the time to five minutes or something similar.
So in the case of helpdesk wanting to coroborate an issue, or guide the user while on the phone, this would then be possible for that amount of time. If helpdesk didn't need to access the account, impersonate gets deactivated for that user again after the time limit has been reached. 

- There also should be a message in the notification area that informs the user about who has used this app on his account for how long. Maybe even an option to send this info via E-Mail to the user, and of course log it in the activity stream of the user.

- Once the impersonation is complete, and the helpdesk has logged out, impersonation gets disabled by that user automatically again.

This would work great with #50

**Comment by wiswedel** [comment]
> There also should be a message in the notification area that informs the user about who has used this app on his account for how long. Maybe even an option to send this info via E-Mail to the user, and of course log it in the activity stream of the user.

I moved this point into a separate issue (#80) since it's the basic first step, while the other two are a more detailed feature.
